tests/by-util/test_fmt.rs

```diff
@@ -130,6 +130,15 @@ fn test_fmt_goal_bigger_than_default_width_of_75() {
     }
 }
 
+#[test]
+fn test_fmt_goal_too_small_to_check_negative_minlength() {
+    for param in ["-g", "--goal"] {
+        new_ucmd!()
+            .args(&["one-word-per-line.txt", "--width=75", param, "10"])
+            .succeeds();
```

Comment:
Also check that the output is sane:
```suggestion
            .succeeds()
            .stdout_is("this is a file with one word per line\n");
```